CHANGELOG.md

```diff
@@ -9,7 +9,10 @@
 
 ## Unreleased
 
-* Remove the inverse flag for contents list component ([PR #1090](https://github.com/alphagov/govuk_publishing_components/pull/1090))
+## 20.0.0
```

Comment:
Version number needs changing in 3 files, see https://github.com/alphagov/govuk_publishing_components/pull/1082/files

Reply:
thanks! done :)